e2e/cypress/integration/ui/settings/team_add_users.spec.ts

```diff
@@ -52,6 +50,7 @@ describe('team add users', () => {
     // the whole app has to reload, slowing down the test and causing timeouts
```

Comment:
BTW, did you happen to try what this TODO comment suggested?

Reply:
@msorens I did, without much luck. If only done in the before, it seemed that the store wasn't properly updated. I also considered clicking the "Add Users" button every time, although that seemed slightly out of scope (is that then a "Team Details" test? 🤔).